### PR TITLE
Integrate user profile with backend

### PR DESCRIPTION
**Pull request by munteanumarco**

### CODE REVIEW

**Review by gheorghe-sava — APPROVED** [review]

File: event-planner-frontend/src/features/profile/ProfilePage.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+function ProfilePage() {
+  const isSuccess = useSelector(selectProfileIsSuccess);
+  const isDone = useSelector(selectProfileIsDone);
+  const toast = useToast();
+
+  useEffect(() => {
+    if (isSuccess) {
+      toast({
+        title: 'Success',
+        description: 'You have successfully updated your profile.',
```
Comment (HellHound97) [inline event-planner-frontend/src/features/profile/ProfilePage.tsx]: Why the message did not come from backend?

File: event-planner-frontend/src/features/profile/ProfilePage.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import { useEffect } from 'react';
+
+function ProfilePage() {
+  const isSuccess = useSelector(selectProfileIsSuccess);
+  const isDone = useSelector(selectProfileIsDone);
+  const toast = useToast();
+
+  useEffect(() => {
+    if (isSuccess) {
+      toast({
```
Comment (HellHound97) [inline event-planner-frontend/src/features/profile/ProfilePage.tsx]: Create a service for error handling on frontend

File: event-planner-frontend/src/features/profile/api/dtos.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  firstName?: string;
+  lastName?: string;
+  dateOfBirth?: Date;
+  country?: string;
+  county?: string;
+  city?: string;
+  profilePhoto?: string;
+};
+
+export const defaultProfile: GetUserProfileDto = {
```
Comment (HellHound97) [inline event-planner-frontend/src/features/profile/api/dtos.ts]: remove this, in dto should only be the type that you need

File: event-planner-frontend/src/features/profile/ProfilePage.tsx (hunk 1 above)
Comment (munteanumarco) [reply event-planner-frontend/src/features/profile/ProfilePage.tsx]: Because in this case the backend returns the updated user, no message

**Pull request merged**